src/components/SearchResults/SearchResults.js

```diff
@@ -374,7 +380,7 @@ const SearchResults = () => {
       <SearchResultsContainer>
         {/* new header for search bar */}
         <HeaderContainer>
-          <H1 style={{ color: '#00684A', paddingBottom: '40px' }}> Search Results</H1>
+          <H1> Search Results</H1>
```

Comment:
(Super duper nit) My imaginary OCD is shouting at me here

```suggestion
          <H1>Search Results</H1>
```

Reply:
your OCD is appreciated! removed